app/controllers/concerns/paginates_answers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module PaginatesAnswers
+  def paginate_answers
+    answer_ids = @answers.map(&:id)
+    answer_ids += @pinned_answers.pluck(:id) if @pinned_answers.present?
+    @answers_last_id = answer_ids.min
+    @more_data_available = !@user.cursored_answers(last_id: @answers_last_id, size: 1).count.zero?
```

Comment:
I get a `NoMethodError` here on `QuestionController#show`, please fix this and add a test:
```
NoMethodError - undefined method `cursored_answers' for nil:NilClass
    @more_data_available = !@user.cursored_answers(last_id: @answers_last_id, size: 1).count.zero?
                                 ^^^^^^^^^^^^^^^^^:
  app/controllers/concerns/paginates_answers.rb:8:in `paginate_answers'
  app/controllers/question_controller.rb:7:in `show'
  app/controllers/application_controller.rb:23:in `switch_locale'
```